lib/ex_format/lines.ex

```diff
@@ -0,0 +1,15 @@
+defmodule ExFormat.Lines do
+  def initialize_lines_store(string) do
+  	lines = String.split(string, "\n")
+    Agent.start_link(fn -> %{} end, name: :lines)
+    for {line, i} <- Enum.with_index(lines) do
+      update_line(i + 1, String.trim(line))
+    end
```

Comment:
Great, thanks for the suggestion. For some reason, it doesn't work (20+ test cases failing) when I populate the `lines` data structure before `start_link`ing it. I pushed my changes but I commented it out. Still trying to figure it out for now. 😂